src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -65,89 +67,93 @@ import dashboardsTimePicker from 'images/dashboards_screenshot-crop_time-picker.
+    
+  </Collapser>
 <Collapser
     id="dash-edit"
     title="Edit your dashboard"
   >
-    You can edit a dashboard if you have [permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions) to do it.
+    You can edit a dashboard if you've [permissions](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-permissions) to do it.
 
     Use the edit <Icon name="fe-edit-2"/> button to:
 
     * Copy the dashboard's permalink.
-    * Rename your dashboard. Names are searchable, so we recommend giving it a meaningful name.
-    * Create new content by clicking the **Add widget** button. Add a new chart using the [query builder](/docs/chart-builder/use-chart-builder/get-started/introduction-chart-builder), or add text, images, or links using our [Markdown editor](#markdown).
-    * Resize and rearrange charts. You can move any chart and put it anywhere in the dashboard so the layout you set fits your preferences: place your more relevant charts on top, or drop less used charts in a corner. You can set up to 12 columns of charts.
+    * Rename your dashboard. [Names are searchable](#dashboards-search), so we recommend giving a meaningful name using words that will help you locate your dashboard easily.
+    * Create new content by clicking the **+ Add widget** button. Add a new chart using the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/), or add text, images, or links using our [Markdown editor](#markdown).
+    * Resize and rearrange [charts](/docs/query-your-data/explore-query-data/use-charts/use-your-charts/). You can move any chart and put it anywhere in the dashboard so the layout you set fits your preferences: place your more relevant charts on top, or drop less used charts in a corner. You can set up to 12 columns of charts.
   </Collapser>
+
   <Collapser
-    id="dash-visual-tv"
-    title="TV mode"
+    id="dash-export"
+    title="Export your dashboard"
   >
-  You can enable a full-screen TV mode that optimizes the dashboard for display on a television screen.
-
-  There're two ways to turn on TV mode:
+    You can export your dashboard as a pdf file. Click the <Icon name="fe-more-horizontal"/> icon on the right corner and select **Export dashboard as PDF**. A new tab on your browser opens with your PDF.
 
-      * When viewing a dashboard, select the <Icon name="fe-tv"/> icon at the top right.
-
-      * Add this parameter to a dashboard page URL: `&platform[tvMode]=true`
-
-
-  To configure TV mode, from a dashboard, click the <Icon name="fe-more-horizontal"/> icon on the right corner and select **Settings**. Under the TV mode section, there're these options:
-
-      * **Display the name of the dashboard**. Turning off the dashboard name gives the dashboard charts more space on the screen.
+    Exporting to PDF or as an image has certain limitations depending on the chart type.
```

Comment:
What if you wrote something more like:

Exporting to PDF or as an image changes the display depending on the chart type.

Reply:
Much better!!! Thank you, @homelessbirds!